### PR TITLE
Fix bug

### PR DESCRIPTION
**Pull request by gabaldon**
Fix ListAddresses compo and pagination.

### CODE REVIEW

File: src/components/Receive.vue [hunk 1]
```diff
@@ -1,68 +1,36 @@
...
-
-            <div class="submit">
-              <Button class="button" :onClick="generateAddress" type="primary"
-                >Generate address</Button
-              >
-            </div>
-          </div>
+      <div class="wrapper">
+        <div data-test="last-address" id="last-generated-address" class="address">
+          {{ addresses[0].address }}
```
Comment (Tommytrg) [inline src/components/Receive.vue]: Add a computed prop: 'lastAddress' which value is addresses[0].address

File: src/utils.js [hunk 2]
```diff
@@ -52,8 +51,27 @@ export function formatSectionApiErrorsByRoute(routeName, errorsMap, apiErrors) {
...
-        ${apiErrors[errorName].message}.
-        ${JSON.stringify(apiErrors[errorName].data)}
+      ${apiErrors[errorName].message}.
+      ${JSON.stringify(apiErrors[errorName].data)}
       `,
     }))
 }
+
+export function copyToClipboard(id) {
+  var textToCopy = document.getElementById(id)
```
Comment (Tommytrg) [inline src/utils.js]: Avoid 'var' usage

File: src/store/wallet.js [hunk 3]
```diff
@@ -207,6 +222,9 @@ export default {
         seedSource: params.sourceType,
         password: params.password,
       })
+
```
Comment (Tommytrg) [inline src/store/wallet.js]: Remove console.log

File: src/store/wallet.js [hunk 4]
```diff
@@ -87,6 +91,13 @@ export default {
       }
       state.addresses = Array.from(addressesMap.values())
     },
+    // setPages(state, { addresses }) {
```
Comment (Tommytrg) [inline src/store/wallet.js]: Remove comment

File: src/components/Wallet.vue [hunk 5]
```diff
@@ -44,24 +38,23 @@ export default {
...
-          // receive: ['receiveError'],
-          send: ['createVTTError'],
-        },
-        {
-          getTransactionsError: this.getTransactionsError,
-          createVTTError: this.createVTTError,
-        }
-      )
-    },
+    // errors() {
```
Comment (Tommytrg) [inline src/components/Wallet.vue]: Remove comment

File: src/components/Wallet.vue [hunk 6]
```diff
@@ -1,33 +1,27 @@
 <template>
   <div>
     <TopBar :tabs="tabs" />
-    <Alert
+    <!-- <Alert
```
Comment (Tommytrg) [inline src/components/Wallet.vue]: Remove comment

File: src/utils.js (hunk 2 above)
Comment (aesedepece) [reply src/utils.js]: Smells like copy-paste :stuck_out_tongue:

File: src/utils.js [hunk 7]
```diff
@@ -52,8 +51,27 @@ export function formatSectionApiErrorsByRoute(routeName, errorsMap, apiErrors) {
...
       description: `
-        ${apiErrors[errorName].message}.
-        ${JSON.stringify(apiErrors[errorName].data)}
+      ${apiErrors[errorName].message}.
+      ${JSON.stringify(apiErrors[errorName].data)}
       `,
     }))
 }
+
+export function copyToClipboard(id) {
```
Comment (Tommytrg) [inline src/utils.js]: Can this function return a boolean value to kwnow if it copied the value?

File: src/components/Transactions.vue [hunk 8]
```diff
@@ -1,26 +1,86 @@
 <template>
-  <div class="transactions">
+  <div data-test="transactions" class="transactions">
```
Comment (Tommytrg) [inline src/components/Transactions.vue]: Do you need this data-test?

File: src/components/Alert.vue [hunk 9]
```diff
@@ -11,6 +11,8 @@
 </template>
 
 <script>
+import { mapState } from 'vuex'
```
Comment (Tommytrg) [inline src/components/Alert.vue]: Are you using this mapState?

File: src/components/Transactions.vue (hunk 8 above)
Comment (gabaldon) [reply src/components/Transactions.vue]: I need it for testing

**Pull request merged**